go.mod

```diff
@@ -15,6 +15,7 @@ require (
 	github.com/moby/term v0.0.0-20210619224110-3f7ff695adc6 // indirect
 	github.com/ory/dockertest/v3 v3.7.0
 	github.com/peggyjv/gravity-bridge/module v0.2.13
+	github.com/regen-network/cosmos-proto v0.3.1 // indirect
```

Comment:
Normally wouldn't ask for such a small detail, but this go.mod line was added when I ran make proto-gen. Is it the right one or is it something that we want a replace directive for?

Reply:
I think this is OK